emulatedtests/utils/docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+name: microsoft-azure-eventhubs
+services:
+  # Service for the Event Hubs Emulator
+  emulator:
+    container_name: "eventhubs-emulator"
```

Comment:
Is there a reason why EventHub tests aren't being run? If so, we can remove the EventHub related setup from this file

Reply:
could be used for next pr to add eventhub